python/lsst/fgcmcal/fgcmOutputProducts.py

```diff
@@ -73,7 +73,7 @@ class FgcmOutputProductsConfig(pexConfig.Config):
     doReferenceCalibration = pexConfig.Field(
```

Comment:
What's the relationship between this `doReferenceCalibration` and the one in fgcmFitCycle?

Reply:
I've improved the doc string: "Transfer 'absolute' calibration from reference catalog? This afterburner step is unnecessary if reference stars were used in the full fit in FgcmFitCycleTask."
I've also added a warning in `FgcmOutputProductsTask` if both are set: "doReferenceCalibration is set, and is possibly redundant with fitCycleConfig.doReferenceCalibration".  (It's not an error, it's just a warning).  Does that make sense?